src/Forms/DatabaseFormFactory.php

```diff
@@ -107,6 +107,22 @@ public function createSelectRollGroup($name, $gibbonSchoolYearID, $all = false)
             return $this->createSelect($name)->fromArray(array("*" => "All"))->fromResults($results)->placeholder();
     }
 
+    public function createSelectHouse($name)
+    {
+        $data = [];
+        $sql = "SELECT gibbonHouseID as value, name FROM gibbonHouse;";
+        try
+        {
+            $results = $this->pdo->executeQuery($data, $sql);
```

Comment:
Hi Jim! There's a couple outstanding PRs from the tail end of v18. It'd be great to get them merged into v19. In this case, its a tiny change, but the whole try/catch can be replaced with a `$results = $this->pdo->select($sql);` line.